creme/mobile/setting_keys.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+from django.utils.translation import gettext_lazy as _
+
+from creme.creme_core.core.setting_key import SettingKey
+
+# google
+#  - search : https://www.google.com/maps/?q={search}
+#  - place : http://maps.google.com/maps/place/{lat},{lng}
+# openstreetmap
+#  - search : https://www.openstreetmap.org/search?query={search}
+#  - place : http://www.openstreetmap.org#map=18/{lat}/{lng}
```

Comment:
SettingValue are for end users ; so put this examples if the description (end won't read this source code), or use settings.py

Reply:
It was only a reminder for me, I should add this in README. I tried to set in the key description, but multiline seems not working.